erdpy/projects/report/data/option_results.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, List, Optional
+
+from erdpy.projects.report.data.common import first_non_none, merge_values_by_key
+from erdpy.projects.report.format.change_type import ChangeType
+from erdpy.projects.report.format.format_options import FormatOptions
+
+
+class OptionResults:
+    def __init__(self, option_name: str, results: List[str]) -> None:
+        self.option_name = option_name
+        self.results = results
+
+
+    def to_json(self) -> Any:
+        return {
+            'option_name': self.option_name,
+            'results': self.results
+        }
+
+
+    @staticmethod
+    def from_json(json: Any) -> 'OptionResults':
+        return OptionResults(json['option_name'], json['results'])
+
+
+    def results_to_markdown(self, format_options: FormatOptions) -> str:
+        separator = ' :arrow_right: ' if format_options.github_flavor else ' -> '
+        change_type = self.classify_changes()
+        display_results = prepare_results_for_markdown(self.results)
+        if change_type == ChangeType.NONE:
+            return display_results[0]
+        else:
+            return separator.join(display_results) + ' ' + change_type.to_markdown(format_options)
+    
+
+    def classify_changes(self) -> ChangeType:
```

Comment:
Perhaps document this a little bit? (e.g. _used when comparing two or more reports ... and in the end ... to draw little icons informing the user about the changes ..._; sorry if I did not understand well).